### PR TITLE
Add instance info to detached nodes

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: upup/pkg/fi/cloudup/awsup/aws_cloud.go [hunk 1]
```diff
@@ -787,18 +787,9 @@ func awsBuildCloudInstanceGroup(c AWSCloud, cluster *kops.Cluster, ig *kops.Inst
...
-			key := aws.StringValue(tag.Key)
-			if !strings.HasPrefix(key, TagNameRolePrefix) {
-				continue
-			}
-			role := strings.TrimPrefix(key, TagNameRolePrefix)
-			cm.Roles = append(cm.Roles, role)
-			cm.PrivateIP = aws.StringValue(instance.PrivateIpAddress)
-		}
+
+		addCloudInstanceData(cm, instance)
```
Comment (hakman) [inline upup/pkg/fi/cloudup/awsup/aws_cloud.go:792]:
```suggestion
		addCloudInstanceData(cm, instances[id])
```

**Review by hakman — CHANGES_REQUESTED** [review]

File: upup/pkg/fi/cloudup/awsup/aws_cloud.go (hunk 1 above)
Comment (olemarkus) [reply upup/pkg/fi/cloudup/awsup/aws_cloud.go]: like this?

Comment (hakman) [reply upup/pkg/fi/cloudup/awsup/aws_cloud.go]: yes

**Pull request merged**